validator/core/constants.py

```diff
@@ -30,12 +28,12 @@
 SUBMISSION_ENDPOINT = "/get_latest_model_submission/"
 
 # to update when live
-#PROD_CONTENT_BASE_URL = "https://content.gradients.io"
+# PROD_CONTENT_BASE_URL = "https://content.gradients.io"
 DEV_CONTENT_BASE_URL = "https://dev.content.gradients.io"
 PROD_CONTENT_BASE_URL = "https://dev.content.gradients.io"
 
 # 241 is testnet
-CONTENT_BASE_URL = DEV_CONTENT_BASE_URL # if NETUID == 241 else PROD_CONTENT_BASE_URL
+CONTENT_BASE_URL = DEV_CONTENT_BASE_URL  # if NETUID == 241 else PROD_CONTENT_BASE_URL
```

Comment:
Please uncomment this, and in future, dont leave this without a # TODO:
```suggestion
CONTENT_BASE_URL = DEV_CONTENT_BASE_URL  # if NETUID == 241 else PROD_CONTENT_BASE_URL
```